packages/mermaid/src/config.type.ts

```diff
@@ -1087,7 +1087,7 @@ export interface SequenceDiagramConfig extends BaseDiagramConfig {
    */
   diagramMarginX?: number;
   /**
-   * Margin to the over and under the sequence diagram
+   * Margin to the top and bottom of the sequence diagram
```

Comment:
This is a generated file, please modify the original one: [`packages/mermaid/src/schemas/config.schema.yaml`](https://github.com/mermaid-js/mermaid/blob/6e51f8fd9840a788bcca1f2cd2aabd78bcf3b74a/packages/mermaid/src/schemas/config.schema.yaml#L1614)

Reply:
Oop rookie mistake